README.md

```diff
@@ -196,9 +196,11 @@ Witness-oriented programming was explored in the following episodes:
   - [Episode 37](https://www.pointfree.co/episodes/ep37-protocol-oriented-library-design-part-1): Protocol-Oriented Library Design: Part 1
   - [Episode 38](https://www.pointfree.co/episodes/ep38-protocol-oriented-library-design-part-2): Protocol-Oriented Library Design: Part 2
   - [Episode 39](https://www.pointfree.co/episodes/ep39-witness-oriented-library-design): Witness-Oriented Library Design
+  - [Episode 40](https://www.pointfree.co/episodes/ep40-async-functional-refactoring): Async Functional Refactoring
+  - [Episode 41](https://www.pointfree.co/episodes/ep41-a-tour-of-snapshot-testing): A Tour of Snapshot Testing
```

Comment:
wanna do a 🆓 thing here too? 

I was even thinking we could specifically call out this episode somehow since it's free and shows people how to use the library. maybe even move it to the top. we can do that later though.